salt/beacons/inotify.py

```diff
@@ -117,6 +117,10 @@ def validate(config):
         if 'files' not in _config:
             return False, 'Configuration for inotify beacon must include files.'
         else:
+            if not isinstance(_config['files'], dict):
+                return False, ('Configuration for inotify beacon invalid, '
+                               'files must be a dict.')
+
             for path in _config.get('files'):
 
                 if not isinstance(_config['files'][path], dict):
```

Comment:
`Configuration for inotify beacon must be a list of dictionaries.`

This error is not correct, it must be a dict of dicts (which this PR validates).